### DISCUSSION TITLE
opendtu

### DISCUSSION THREAD

**Discussion by jacques3629** [tbnobody/OpenDTU#1451] [General]
Bonjour 

avec mqtt  je ne vois que des commandes info , existe-t-il une commande pour réduire la puissance des onduleurs

merci

with mqtt I only see info commands, is there a command to reduce the power of the inverters

**Comment by stefan123t** [comment]
Salut Jacques et bienvenue,

les commandes limite de puissance ce sont documenteé à https://github.com/tbnobody/OpenDTU/blob/master/docs/MQTT_Topics.md#inverter-limit-specific-topics

power limit commands are documented at the above link.

**Comment by jacques3629** [comment]
Merci beaucoup tout est réglé maintenant pour Moi
